### PR TITLE
Fix pathname encoding, #544

### LINKED ISSUE #544: App can't run from a path that has `#` in it
e.g. `c:\electron\#net\yakyak`

### PR DESCRIPTION
**Pull request by refack**
Fix #544

### CODE REVIEW

File: src/main.coffee [hunk 1]
```diff
@@ -114,7 +118,7 @@ app.on 'ready', ->
         height: 590
         "min-width": 620
         "min-height": 420
-        icon: path.join __dirname, 'icons', 'icon@32.png'
+        icon: path.join YAKYAK_ROOT_DIR, 'icons', 'icon@32.png'
```
Comment (davibe) [inline src/main.coffee]: are you sure about this ? is icon a plain path or an url ? I believe a plain path should not be encoded.

Comment (refack) [inline src/main.coffee]: Investigating...

**Comment by refack** [comment]
Yep, you were right, too much encoding.
Now it works, but I'm not sure about the `toast` notification (they stopped working either way)

**Comment by davibe** [comment]
Maybe we manage `toast` notifications in a separate issue/pr ?

**Comment by averissimo** [comment]
global options can be retrieved on the client side, i think `force_close` is one of them

**Comment by refack** [comment]
I've seen it, but thought I killed it (client-side `undefined`). 1st try definatly fixed it, and the 2nd try probably regressed it back in.
I think the `toast` issue is related to the custom sound. Will spin off a new issue.

**Comment by refack** [comment]
Can I get a hint as to what's the entrypoint to the client process?

**Comment by davibe** [comment]
in main.coffee we create a BrowserWindow. We actually create one for login and then one for the main window. Anyway, the mainWindow will load `ui/index.html` which in turn loads the compiled `ui/app.coffee` so this last one is the entry point you want, i think. In that file we already use `remote.getGlobal`. Everything that lives in `ui/` is about the renderer process.

**Pull request merged**